compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -213,13 +214,21 @@ class ComposePanel @ExperimentalComposeUiApi constructor(
             _composeContainer?.windowContainer = value
         }
 
-    override fun add(component: Component): Component {
-        return super.add(component)
-    }
+    /**
+     * Returns the [SemanticsOwner]s corresponding to the roots of the semantics trees in this
+     * [ComposePanel].
+     */
+    @ExperimentalComposeUiApi
```

Comment:
Please make an issue with target 1.12 for stabilization

Reply:
We have older experimental APIs in these classes. Let's create task for a bit larger scope, not only for this field

Reply:
It is preferable to separate them. The new API should be stabilized in 1.12. The old ones, probably in 1.10